README.md

```diff
@@ -147,6 +147,10 @@ Examples:
   
 [configure-jvm]: https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory
 
+## Disabling encryption when releasing a plugin
+
+When performing a release via `gradle publish`, gradle will automatically tried to upload artifacts with SHA 256 and 512 encryption. This is not currently supported in the public artifact repository for Jenkins.  To disable this, you can pass a command line argument `gradle publish -Dorg.gradle.internal.publish.checksums.insecure` or include a `gradle.properties` file with the line `org.gradle.internal.publish.checksums.insecure=true`. 
```

Comment:
SHA-256/512 aren't encryption.

Reply:
This was updated as part of https://github.com/jenkinsci/gradle-jpi-plugin/commit/3f224576d7e81762391677425782599210744328